hnn_core/neuron.py

```diff
@@ -129,27 +126,41 @@ def simulation_time():
     return dpl
 
 
+def _is_loaded_mechanisms():
+    # copied from:
+    # https://www.neuron.yale.edu/neuron/static/py_doc/modelspec/programmatic/mechtype.html
+    mt = h.MechanismType(0)
+    mname = h.ref('')
+    mnames = list()
+    for i in range(mt.count()):
+        mt.select(i)
+        mt.selected(mname)
+        mnames.append(mname[0])
+    if 'hh2' not in mnames:
```

Comment:
True, it's not a very solid check. I am feeling lazy to change this now. I do expect some of the tests to fail though if someone changes the name of a mod file, so hopefully it will be caught early and can be improved at that point.

Reply:
Fair enough!